test/models/post_test.rb

```diff
@@ -1,7 +1,24 @@
 require 'test_helper'
 
 class PostTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test "defaults opengraph fields to nil when no links in body text" do
+    post = FactoryGirl.create(:post)
+    assert_nil post.og_url
+    assert_nil post.og_title
+    assert_nil post.og_description
+    assert_nil post.og_image
+  end
+
+  test "grabs opengraph data from site url" do
+    FakeWeb.register_uri(:any, 'http://www.thefirehoseproject.com', :response => File.join('test/fixtures/og_response.json'))    
```

Comment:
This is such a nitpick.  I'm sorry.

The file `og_response` isn't in JSON format.  The extension should probably be `.html` or maybe `.http` since it includes both the HTML of the page and the HTTP headers.

Make that one minor tweak and I'll press the merge button.  :+1: